kyaml/fieldmeta/fieldmeta.go

```diff
@@ -76,6 +81,53 @@ func (fm *FieldMeta) Read(n *yaml.RNode) error {
 	return nil
 }
 
+// processShortHand parses the comment for short hand ref and loads schema to fm
+func (fm *FieldMeta) processShortHand(comment string) bool {
```

Comment:
We should document the meaning of the return value.

Reply:
Is it always safe to just return false whenever we encounter an error?

Reply:
Yes. We should not throw error in any case as the comment might not be related to kpt/setters. We do the best to parse it and see if it is setter ref and continue if it is not.

Reply:
Updated the comment.